src/main/java/org/threadly/concurrent/NoThreadScheduler.java

```diff
@@ -27,14 +26,12 @@
  * @author jent - Mike Jensen
  * @since 2.0.0
  */
-public class NoThreadScheduler extends AbstractSubmitterScheduler 
-                               implements SchedulerService {
+public class NoThreadScheduler extends AbstractPriorityScheduler {
   protected static final int QUEUE_FRONT_PADDING = 0;
   protected static final int QUEUE_REAR_PADDING = 2;
```

Comment:
Do we still need these since they are the same in the Abstract class?  

It might be worth doing this differently as these values get a little confusing with how inheritance is going around, would the QueueSet object use values in the NoThreadScheduler or the AbstractPriorityScheduler?  If memory serves statics work a little wonky with this.